src/platforms/node/guides/aws-lambda/index.mdx

```diff
@@ -16,37 +19,60 @@ $ npm install --save @sentry/node
 $ yarn add @sentry/node
 ```
 
-To set up Sentry error logging for a Lambda Function, build a wrapper:
+You can use the AWS Lambda integration for the Node SDK like this:
 
 ```javascript
-"use strict";
+const Sentry = require("@sentry/serverless");
+
+Sentry.init({
+  dsn:
+    "___PUBLIC_DSN___",
+});
+
+const myAsyncHandler = async (event, context, callback) => {
+  //Your handler code
+};
+
+exports.handler = Sentry.AWSLambda.wrapHandler(myAsyncHandler);
+```
+
+<!-- TODO-ADD-VERIFICATION-EXAMPLE -->
 
-const Sentry = require("@sentry/node");
+## Timeout Warning
 
+Sentry reports timeout warning when the function is within 500ms of it's execution time. You can turn off timeout warnings by setting `captureTimeoutWarning` to `false` during initiatlization. To change timeout warning limit, assign a numeric value (in ms) to `timeoutWarning`.
```

Comment:
This is now how options are set for handlers.

```js
Sentry.AWSLambda.wrapHandler(yourHandler, {
  captureTimeoutWarning: false
});
```

So both, description and code examples are incorrect.